infra/aws/terraform/management-account/capa-lfx-mentorship.tf

```diff
@@ -0,0 +1,24 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+module "k8s-infra-capa-lfx-mentee" {
+  source = "../modules/org-account"
+
+  account_name               = "k8s-infra-capa-lfx-mentee"
+  email                      = "k8s-infra-aws-admins+capa-lfx-mentee@kubernetes.io"
```

Comment:
@dims I don't think we need another email account for this.

Reply:
ack